### PR TITLE
[COC-68] Logout Screen

### PR DESCRIPTION
**Pull request by shape-shifters**
roundstyle.xml added for buttons shape and color

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Alaple) [inline .idea/codeStyles/Project.xml]: Please delete this file

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Alaple) [inline .idea/codeStyles/codeStyleConfig.xml]: Please delete this file

**Review by Alaple — CHANGES_REQUESTED** [review]
Please delete this file

**Review by manuelfalonso — CHANGES_REQUESTED** [review]
Falta este texto :D

image

**Review by Alaple — APPROVED** [review]
As manu said, it lack of one text. Regarding that, the PR LGTM

image

**Review by manuelfalonso — APPROVED** [review]
lgtm

**Pull request merged**